native/command.rs

```diff
@@ -1,6 +1,7 @@
 use argon2::{hash_encoded, verify_encoded, Config, ThreadMode, Variant, Version};
 use bytes::Bytes;
-use deno_core::{Buf, CoreOp, ZeroCopyBuf};
+use deno_core::{Buf, Op, ZeroCopyBuf};
+use deno_core::plugin_api::Interface;
```

Comment:
When working in Rust, I usually prefer to group imports.
```suggestion
use deno_core::{Buf, Op, ZeroCopyBuf, plugin_api::Interface};
```

Sorry about this little obsession 😅